index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <h1 id="title" class="title">Software Survey Form</h1>
+  <form id="survey-form">
+    <div class="name">
+      <h3>Name</h3>
+      <label id="name-label" </label>
+      <input id="name" placeholder="Enter Your Name">
+    </div>
+    <div class="email">
+          <h3>Email</h3>
+      <label id="email-label"</label>
```

Comment:
- [ ] You should have mentioned that the `label` element is not working and the `<h3>` tag is redundant & not semantically meaningful to use it as a form element.